source/2019-04-12-the-ember-times-issue-94.md

```diff
@@ -86,13 +86,16 @@ responsive: true
   
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## [User Impersonation in SaaS with Ember.js and Ruby on Rails 🐹](https://codeandtechno.com/posts/user-impersonation-ember-simple-auth-doorkeeper/)
+
+When developing SaaS (software as a service) applications, it is important to be able to log in as one of your users and see exactly what they see. This can help debugging issues and aid your support team in assisting users. 
+
+[@matixmatix](https://github.com/matixmatix) has created an awesome tutorial on how this can be done using Ember.js ([Ember Simple Auth](https://ember-simple-auth.com/) with Ruby on Rails (with [Doorkeeper](https://github.com/doorkeeper-gem/doorkeeper) gem for 0Auth2). 
```

Comment:
@BradLeftley One more correction: "0Auth2" to "OAuth2" (you used the number 0). Thanks.

Reply:
Sorted :)